package.json

```diff
@@ -29,9 +29,12 @@
     "cross-env": "^7.0.2",
     "express": "^4.16.4",
     "helmet": "^3.22.0",
+    "humps": "^2.0.1",
+    "lodash": "^4.17.15",
     "nuxt": "^2.0.0"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-optional-chaining": "^7.9.0",
```

Comment:
為了可以在舊瀏覽器用 optional chaining，所以加了這個 babel plugin。

Reply:
我有疑問，[@babel/plugin-proposal-nullish-coalescing-operator](https://babeljs.io/docs/en/babel-plugin-proposal-nullish-coalescing-operator) 應該也需加上？

Reply:
嗯嗯對，要加。沒想到這也是新的提案。

Reply:
不知道 `babel-preset-env` options 的 corejs proposals 是不是就可以達到

Reply:
[文檔](https://babeljs.io/docs/en/babel-preset-env#corejs)：
> use `corejs: { version: 3, proposals: true }`. This will enable polyfilling of every proposal supported by `core-js`

但從 [core-js](https://github.com/zloirock/core-js) 中似乎沒看到有支援這個 proposal。